src/test/java/com/jwplayer/southpaw/util/ByteArrayTest.java

```diff
@@ -62,11 +62,12 @@ public void testEquals() {
     @Test
     public void testFromBytes() throws Exception {
         byte[] bytes = Hex.decodeHex("01410204d20100".toCharArray());
-        Set<ByteArray> set = new HashSet<>(3);
-        set.add(new ByteArray("A"));
-        set.add(new ByteArray(1234));
-        set.add(new ByteArray(false));
-        assertEquals(set, ByteArray.fromBytes(bytes));
+        ByteArraySet expectedSet = new ByteArraySet();
```

Comment:
hmm @moritastic does it make sense for `ByteArraySet` to be used to test `ByteArray` here?

Reply:
Yeah, it seems like `toBytes` and `fromBytes` should be on `ByteArraySet`. Not sure why I put it on `ByteArray`.